.yamato/ruamel/jobs/abv/abv_all_project_ci_nightly.py

```diff
@@ -4,40 +4,41 @@
 
 class ABV_AllProjectCiNightlyJob():
     
-    def __init__(self, editor, projects, test_platforms, nightly_config, target_branch):
-        if editor["version"] not in nightly_config["allowed_editors"]:
-            raise Exception(f'Tried to construct nightly with PR trigger for version {editor["version"]}')
-        self.job_id = abv_job_id_all_project_ci_nightly(editor["version"])
-        self.yml = self.get_job_definition(editor, projects, test_platforms, nightly_config.get("extra_dependencies",[]), target_branch).get_yml()
+    def __init__(self, editor, test_platforms, nightly_config, target_branch):
```

Comment:
I think `test_platforms` can be safely removed here, it was used in the `for` block in which we appended the smoke tests to the nigthly, but it's not used anymore.

Reply:
Good catch! Removing it